cli/src/main/scala/org/apache/celeborn/cli/master/MasterSubcommandImpl.scala

```diff
@@ -212,6 +214,51 @@ class MasterSubcommandImpl extends MasterSubcommand {
       commonOptions.configName,
       commonOptions.getAuthHeader)
 
+  private[master] def runUpsertDynamicConf: HandleResponse = {
```

Comment:
The following code block is identical to the worker's subcommand implementation. These codes can be reused.

Reply:
Maybe you can extract a common command.